src/orquestra/quantum/circuits/_gates.py

```diff
@@ -425,6 +433,9 @@ def exp(self) -> "Gate":
     def power(self, exponent: float) -> "Gate":
         return Power(self, exponent)
 
+    def __str__(self):
+        return "exp" + POWER_GATE_SYMBOL + str(self.wrapped_gate)
```

Comment:
btw. could we move the definition of `POWER_GATE_SYMBOL` from line 440 to the top of the file?

Reply:
Yup!

Reply:
I think we should move all of the to the top, though it's not a big deal TBH.